analyze_hc_restore.pl

```diff
@@ -1073,14 +1075,18 @@ sub rem_cmd_line_switch
     {
       $optimized_kernel = "1";
     }
-    elsif (($arg eq "-n") || ($arg eq "--gpu-accel"))
+    elsif (($arg eq "-n") || ($arg eq "-n"))
```

Comment:
To do: revert change and update to `--kernel-accel` to coincide with hashcat's help text